### PR TITLE
non-uom data conversion scripts

### PR DESCRIPTION
**Pull request by philblackwood**
Data conversion scripts for non-uom changes. Closes issue #1091.

### CODE REVIEW

**Comment by sa-bpelakh** [comment]
@philblackwood  I will add the bundle drivers directly into this PR, if that's OK. I don't think you need to do the named graph exclusion on the default graph queries because the pattern should already have been transformed there.

File: migration/v13.0/queries/replaceHasCommunicationAddress1.rq [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# replace gist:hasCommunicationAddress in named graphs with gist:hasAddress
```
Comment (rjyounes) [inline migration/v13.0/queries/replaceHasCommunicationAddress1.rq:1]: Phil, can't we include the `hasAddress/hasCommunicationAddress` replacement in the general `renameProperties` files and eliminate the two additional files?

**Review by rjyounes — CHANGES_REQUESTED** [review]

File: migration/v13.0/queries/replaceHasCommunicationAddress1.rq (hunk 1 above)
Comment (philblackwood) [reply migration/v13.0/queries/replaceHasCommunicationAddress1.rq:1]: yes! I will update the renameProperties scripts to include the pair:

oldProperty: gist:hasCommunicationAddress

newProperty: gist:hasAddress.

**Comment by philblackwood** [comment]
From Boris:  I don't think you need to do the named graph exclusion on the default graph queries because the pattern should already have been transformed there."

me: I think it's OK to leave it in. The scenario where it helps is:

   - user runs first script to make changes in named graph, then puts the data conversion aside for a while
   -  new changes are made in the named graph
   -  user returns to data conversion, runs second script which results in named graph triples removed and their replacement incorrectly gets added to the default graph.

I looked at making the change, and it is big enough (and my typing bad enough) to warrant a re-test, which I would prefer not to have to do.

Are you OK with leaving it in?

**Comment by philblackwood** [comment]
@sa-bpelakh "I will add the bundle drivers directly into this PR, if that's OK."

phil: yes, certainly.

**Comment by philblackwood** [comment]
Adding data conversion for deprecating hasSuperCategory and its sub properties.

**Comment by rjyounes** [comment]
@philblackwood @sa-bpelakh Is this ready to go? It needs to be updated with `develop`.

File: migration/v13.0/non-uomDataConversion.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Be sure to read the header of each data conversion script.
+
+## Examples:
+- renameProperties replaces `gist:isGeographicallyContainedIn` with `gist:isGeoContainedIn`
+
+- replaceAddressClasses replaces `gist:EmailAddress` with `gist:ElectronicAddress`
+- replaceAddressClasses replaces `gist:StreetAddress` with `gist:PhysicalAddress`
+
+The different types and uses of addresses can be modeled as categories.
```
Comment (rjyounes) [inline migration/v13.0/non-uomDataConversion.md:17]:
```suggestion
The different types and uses of addresses can be modeled as categories; see [the address migration documentation(AddressMigration.md) as well as the [full documentation of the new address model](/docs/model_documentation/AddressGuidance.md).
```

**Review by rjyounes — CHANGES_REQUESTED** [review]

**Pull request merged**